include/crocoddyl/multibody/contacts/contact-2d.hpp

```diff
@@ -18,6 +17,7 @@
 #include <pinocchio/multibody/data.hpp>
 #include <pinocchio/algorithm/frames.hpp>
 #include <pinocchio/algorithm/kinematics-derivatives.hpp>
+#include "crocoddyl/multibody/fwd.hpp"
```

Comment:
Same here. Why make these unnecessary changes?
fwd is declared earlier, since it contains forward declarations.